test/rtf-test-files/colors/expected-metadata.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
This doesn't match the meta data in the rtf file, please regenerate the test case